tests/trainer/test_trainer_distributed.py

```diff
@@ -173,6 +174,20 @@ def test_trainer(self):
         # successful return here == success - any errors would have caused an error in the sub-call
 
 
+@require_torch_multi_hpu
+class TestTrainerDistributedHPU(TestCasePlus):
+    def test_trainer(self):
+        distributed_args = f"""--nproc_per_node={torch.hpu.device_count()}
+            --master_port={get_torch_dist_unique_port()}
+            {self.test_file_dir}/test_trainer_distributed.py
+        """.split()
+        output_dir = self.get_auto_remove_tmp_dir()
+        args = f"--output_dir {output_dir}".split()
+        cmd = ["torchrun"] + distributed_args + args
+        execute_subprocess_async(cmd, env=self.get_env())
+        # successful return here == success - any errors would have caused an error in the sub-call
```

Comment:
the tests in this file are implemented for each device separately, but can be grouped in one, is it intended ?